xsl/webhelp-common.xsl

```diff
@@ -639,9 +656,37 @@ border: none; background: none; font-weight: none; color: none; }
     <xsl:template name="webhelpheader.logo">
 	<a href="index.html">
 	<img style='margin-right: 2px; height: 59px; padding-right: 25px; padding-top: 8px' align="right"
-	    src='{$webhelp.common.dir}images/logo.png' alt="{$brandname} Documentation"/>
+	    src='{$webhelp.common.dir}images/logo_Rudder.png' alt="{$brandname} Documentation"/>
 	</a>
     </xsl:template>
+    
+    
+    <xsl:template name="otherlinks">
+    <div id="otherlinks">
+      <span>Resources: <strong>User manual</strong> |
+                             <a href="http://www.rudder-project.org/foswiki/bin/view/System/Documentation:ChangeLog211">Changelog</a> | 
```

Comment:
This is a great idea! However, we *know* that we want to change the URLs for these changelogs, so before merging this PR and sending these URLs out as static code to Rudder installations everywhere, we should address this.

I propose a simple approach: we make a memorable, simple URL on rudder-project.org/changelog-<version> (same format at /doc-<version> and same shortcuts such as /doc, /doc-latest, etc), and use Apache rewrite rules to make sure that URL always goes to the right page. That way, we can change the actual URL no problem, anytime.

I've added the rewrite rules already, can you adjust the PR?